packages/web-components/src/components/footer/footer-composite.ts

```diff
@@ -130,8 +130,13 @@ class DDSFooterComposite extends ModalRenderMixin(HybridRenderMixin(HostListener
   @property()
   language?: string;
 
-  @property({ attribute: false })
-  langList?;
+  /**
+   * Placeholder list of languages to populate language selector
+   *
+   * @internal
+   */
+  @property({ attribute: 'lang-list' })
+  langList?: string;
```

Comment:
Please see my comment to `JSON.parse()` code for more details:

```suggestion
  @property({ attribute: false })
  langList?: string[];
```

Reply:
Setting it to `string[]` makes Typescript complain when trying to access its contents (`text` does not exist on type String), though it works anyway.

Changing it to `any[]` gets rid of this error.